ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_core.cc

```diff
@@ -87,59 +119,78 @@ TFCore::getModelPath ()
   return model_path;
 }
 
+/**
+ * @brief	the definition of a deallocator method
+ */
+static void
+DeallocateBuffer (void* data, size_t t) {
+  std::free (data);
+}
+
 /**
  * @brief	load the tf model
  * @note	the model will be loaded
  * @return 0 if OK. non-zero if error.
- *        -1 if the modelfile is not valid(or not exist).
- *        -2 if the pb file is not loaded.
- *        -3 if the input properties are different with model.
- *        -4 if the Tensorflow session is not initialized.
- *        -5 if the Tensorflow session is not created.
+ *        -1 if the pb file is not loaded.
+ *        -2 if importing graph is failed.
+ *        -3 if the Tensorflow session is not created.
  */
 int
 TFCore::loadModel ()
 {
 #if (DBG)
   gint64 start_time = g_get_real_time ();
 #endif
+  g_assert (model_path != nullptr);
+  const auto f = std::fopen (model_path, "rb");
```

Comment:
Line 145 ~ 160
Plz consider to use another functions (like g_file_get_contents ())

Security issue, (this will make error while processing the internal security task)
https://wiki.sei.cmu.edu/confluence/display/c/MSC24-C.+Do+not+use+deprecated+or+obsolescent+functions

MSC24-C. Do not use deprecated or obsolescent functions   https://wiki.sei.cmu.edu/confluence/display/c/MSC24-C.+Do+not+use+deprecated+or+obsolescent+functions | MSC24-C. Do not use deprecated or obsolescent functions   https://wiki.sei.cmu.edu/confluence/display/c/MSC24-C.+Do+not+use+deprecated+or+obsolescent+functions
-- | --
MSC24-C. Do not use deprecated or obsolescent functions   https://wiki.sei.cmu.edu/confluence/display/c/MSC24-C.+Do+not+use+deprecated+or+obsolescent+functions

Reply:
it's applied with `g_file_get_contents()`.